openeogeotrellis/deploy/batch_job.py

```diff
@@ -120,6 +125,8 @@ def extract_result_metadata(tracer: DryRunDataTracer) -> dict:
             logger.warning("Multiple aggregate_spatial geometries: {c}".format(c=len(aggregate_spatial_geometries)))
         agg_geometry = aggregate_spatial_geometries[0]
         if isinstance(agg_geometry, BaseGeometry):
+            # We only allow EPSG:4326 for the BaseGeometry case.
+            bbox_crs = "EPSG:4326"
             bbox = agg_geometry.bounds
             geometry = mapping(agg_geometry)
             area = area_in_square_meters(agg_geometry, "EPSG:4326")
```

Comment:
Would replace `"EPSG:4326"` with `bbox_crs`.

Reply:
I missed that one. 
Done.